pom.xml

```diff
@@ -36,14 +36,14 @@
     </developers>
 
     <properties>
-        <revision>2.4-SNAPSHOT</revision>
+        <revision>2.5-SNAPSHOT</revision>
 
         <maven.compiler.release>17</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <quarkus.version>3.6.0</quarkus.version>
-        <hilla.version>2.4-SNAPSHOT</hilla.version>
+        <hilla.version>2.5-SNAPSHOT</hilla.version>
```

Comment:
All places adjusted?

Reply:
tests modules inherit from parent pom, so they use the same property